### ISSUE TITLE
Constant, slight VU meter noise

### ISSUE THREAD

**Issue by danboid** [FigBug/SID#2]
I have tried both the official Tracktion SID binary and I have also built SID from source and I see the same issue under both REAPER and Ardour in that when SID is added to a midi track, the VU meter for that track constantly shows some slight noise - bouncing around the -45dB mark - even when no MIDI data is being sent to the track. Is this to be expected?

PAPU and the other socalabs plugins do not have this issue.

**Comment by FigBug** [comment]
Yes, it’s an accurate emulation of the C64 that has a bit of 0 offset.


On September 28, 2017 at 11:14:08 AM, danboid (notifications@github.com)
wrote:

I have tried both the official Tracktion SID binary and I have also built
SID from source and I see the same issue under both REAPER and Ardour in
that when SID is added to a midi track, the VU meter for that track
constantly shows some slight noise - bouncing around the -45dB mark - even
when no MIDI data is being sent to the track. Is this to be expected?

PAPU and the other socalabs plugins do not have this issue.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/FigBug/SID/issues/2>, or mute the thread
